qiskit_nature/second_q/operators/_bits_container.py

```diff
@@ -0,0 +1,154 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""A bit-storage container."""
+
+from collections.abc import MutableMapping
+from typing import Generic, TypeVar
+
+# pylint: disable=invalid-name
+T = TypeVar("T")
+
+
+class _BitsContainer(MutableMapping, Generic[T]):
+    """A bit-storage container.
+
+    This is a utility object used during the simplification process of a operators.
+    It manages access to an internal data container, which maps from integers to bytes.
+    Each integer key corresponds to a vibrational mode of an operator term.
+    Each value consists of four bits which encoding for the corresponding index:
+
+        1. if a `+` has been applied
+        2. if a `-` has been applied
+        3. whether a `+` or `-` was applied first
+        4. whether the last applied operator was a `+` or `-`
```

Comment:
```suggestion
    Each value consists of four bits which encoding for the corresponding index:

        1. if a `+` has been applied
        2. if a `-` has been applied
        3. whether a `+` or `-` was applied first
        4. whether the last applied operator was a `+` or `-`.
```

Reply:
Thanks! I will also correct the wording a bit :+1: